_rules/invalid-form-field-value-36b590.md

```diff
@@ -241,4 +241,4 @@ There are no elements with any of the required [semantic roles][semantic role].
 [required context]: https://www.w3.org/TR/wai-aria/#scope
 [semantic role]: #semantic-role 'Definition of semantic role'
 [text]: https://www.w3.org/TR/WCAG21/#dfn-text
-[visible]: #visible 'Definition of visible'
+[visible]: #visible 'Definition of visible'
```

Comment:
This shouldn't change. We don't want this commit message to end in the history of the form-field-value rule.

Reply:
Please fix this.